src/families/tron/bridge/js.js

```diff
@@ -259,67 +298,108 @@ const currencyBridge: CurrencyBridge = {
   scanAccounts
```

Comment:
regarding currencyBridge, it would be beneficial to try to load SRs in a preload and handle it in the hydrate.

Here is what we did on Tezos: https://github.com/LedgerHQ/ledger-live-common/blob/14294a716df5b9fd12984e1c99080d18267af5f9/src/families/tezos/bridge/libcore.js#L244-L260

preload/hydrate is a system that will be used on the UI to load and cache data the currency need (see the types comments)

In Tezos we also had cache layer in bakers.js in order to only load the bakers list once. Different cache strategy can be implemented based on how often you need to reload the data, but the key point is that it will works offline if you have loaded it once (hydrate fills back the previous data).

In Tezos, we need the bakers to effectively display the "current delegation" information, because we reconstruct that data with operations and bakers list.
I don't know how you plan to solve that, but for TRON, I should be able to go to visit my account without having a "loading..." time for the "currently voting to", and that system would address this.

to be clear, preload is called before scanning accounts (at least on mobile/desktop) and also before each sync – but it's called only once per currency.
This system is in place to both minimize the http calls and make everything working offline (at least in read only).